database/migration/databaseMethods.ts

```diff
@@ -166,6 +167,63 @@ const databaseMethods = {
       }
     }
   },
+  migrateTemplateVersions: async () => {
```

Comment:
Quite a substantial migration to move existing templates over to the new versioning schema.